isis/tests/FunctionalTestsIsisImportEis.cpp

```diff
@@ -83,10 +86,94 @@ TEST_F(TempTestingFiles, FunctionalTestIsisImportEisNacFrame){
     End_Object
     End
   )");
-  QString dataFileName = " data/eis2isis/nacFrame/nac000xxx_2022145t000000_0000000001_frame_raw02.xml";
+  QString dataFilePath= " data/isisimport/eispds/nacFrame/nac000xxx_2022145t000000_0000000001_frame_raw02.xml";
+  QString dataFileName = "nac000xxx_2022145t000000_0000000001_frame_raw02.xml";
+  QString imageFilePath = " data/isisimport/eispds/nacFrame/nac000xxx_2022145t000000_0000000001_frame_raw02.img";
+  QString imageFileName = "nac000xxx_2022145t000000_0000000001_frame_raw02.img";
   QString templateFile = "../appdata/import/PDS4/ClipperEIS.tpl";
   QString cubeFileName = tempDir.path() + "/nacFrame.cub";
-  QVector<QString> args = {"from=" + dataFileName, "to=" + cubeFileName, "template=" + templateFile};
+
+  // hard coding these values for now, but we should use some sort of xml parser to grab these values from the labels xml
+  int samples = 3832;
+  int lines = 1335;
+  int bytes = 2;
+
+  // create a temp img file and write data to it
+  QFile tempImgFile(tempDir.path() + "/" + imageFileName);
+  if(!tempImgFile.open(QFile::WriteOnly | QFile::Text)){
+      qDebug() << " Could not open file for writing";
+      return;
+  }
+  QTextStream out(&tempImgFile);
+  char writeToFile[samples * bytes];
+  for(int i=0; i<samples * bytes; i++){
+    writeToFile[i] = '0';
```

Comment:
I don't think we want to write text '0's to the file. Rather just 0 bytes. You should be able to use a a [qDataStream](https://doc.qt.io/qt-6/qdatastream.html) to do this.

Reply:
If you really want to properly test the data ingest, then you could write MSB short ints to the datastream. I don't know if that' actually needed though.